sova/config/db_loader.py

```
@@ -0,0 +1,185 @@
+    for row in rows:
+        try:
+            value = json.loads(row.value)
+        except (json.JSONDecodeError, TypeError):
+            logger.warning("Skipping setting %r: invalid JSON value", row.key)
+            continue
+        _set_nested(nested, row.key, value)
+
+    return nested if nested else None
+
+
+async def save_setting(session: AsyncSession, key: str, value: Any) -> None:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)